assets/js/lib/network/network.test.js

```diff
@@ -188,4 +188,23 @@ describe('networkClient', () => {
       }
     });
   });
+
+  describe('REST operations', () => {
```

Comment:
Is this test really testing anything?

Reply:
Nothing SUMA specific. It's pretty naive I know 😂 I can live also without it.

Reply:
I guess `axiosMock.onPatch()` is doing the heavy-lifting here and validates that the request is a PATCH request. :muscle:

I'm fine with it either way also, but I guess we should also then test the other verbs in `networkClient`, no?

Reply:
The test you were looking for is not here :smile: